build.zig

```diff
@@ -526,9 +525,6 @@ fn addTest(b: *std.Build, lib: *std.Build.Step.Compile,
 fn installHeader(b: *std.Build, lib: *std.Build.Step.Compile,
                  hfile: []const u8) void {
    const inc_path = "include/";
```

Comment:
Is my understanding correct that '/' after "include" is redundant?

Reply:
Yes, the way you simplified it in the end should be sufficient :)